cli/components/selector.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package components
+
+import (
+	"fmt"
+	"os"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/rs/zerolog/log"
+)
+
+// Selector is the interface that items need to implement so that we can select them.
+type Selector interface {
```

Comment:
this one also seems to generic. `CliSelectableItem` maybe?

Reply:
Sure.